### PR TITLE
fix(bake): include account in find packages

### PR DESCRIPTION
**Pull request by emjburns**
Accidentally didn't use account in the query to find an image by package name. That didn't work. Trying again with the right api calls here.

Also removing `titus-cluster` from tag-able things because I forgot to earlier

### CODE REVIEW

File: keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt [hunk 1]
```diff
@@ -77,7 +77,6 @@ class ResourceTagger(
 
   private val taggableResources = listOf(
     "cluster",
-    "titus-cluster", // todo eb: fix to just be cluster
```
Comment (luispollo) [inline keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt]: Unrelated change?

Comment (emjburns) [reply keel-tagging-plugin/src/main/kotlin/com/netflix/spinnaker/keel/tagging/ResourceTagger.kt]: Yes, but I snuck it in. Bad practice.

File: keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt [hunk 2]
```diff
@@ -98,10 +98,10 @@ class ImageService(
    * Each ami must have tags.
    */
   suspend fun getLatestNamedImageWithAllRegionsForAppVersion(appVersion: AppVersion, account: String, regions: List<String>): NamedImage? =
-    cloudDriverService.images(
+    cloudDriverService.namedImages(
       serviceAccount = DEFAULT_SERVICE_ACCOUNT,
-      provider = "aws",
-      name = appVersion.toImageName().replace("~", "_")
+      imageName = appVersion.toImageName().replace("~", "_"),
+      account = account
```
Comment (asher) [inline keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt]: Could this end up being the account we'll want to deploy to instead of the account containing the image?

Comment (emjburns) [reply keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt]: We pass in "test" here (when used in the ImageHandler). Same with the use in the ImageResolver. So I don't think so.
